django_ip_geolocation/backends/base.py

```diff
@@ -16,6 +16,8 @@ def _parse(self):
         raise NotImplementedException()
```

Comment:
Already said, it looks strange to expose underscored method...

Reply:
In python there is no 'standard' for private and protected methods. some agree that underscore methods or attribute are protected, and double underscore prefixed are private. others consider prefixed underscore methods are private.

I think what's more important is to tell developers that this method is internal and must be implemented in order to parse custom json response.